terraform/groups/ecs-service/s3.tf

```diff
@@ -0,0 +1,52 @@
+resource "aws_kms_key" "file_transfer_encryption_key" {
+  description             = "Encrypt user uploaded data for the extensions project."
+  deletion_window_in_days = 30
+}
+
+resource "aws_kms_key" "file_transfer_encryption_key_secure" {
+  description             = "Encrypt user uploaded data for the extensions project."
+  deletion_window_in_days = 30
+}
+
+
+data "aws_kms_alias" "file_transfer_encryption_key_alias_secure" {
+  name          = var.file_transfer_kms_alias_secure
+}
+
+data "aws_kms_alias" "file_transfer_encryption_key_alias" {
+  name          = var.file_transfer_kms_alias
+}
+
+data "aws_s3_bucket" "file_transfer_bucket" {
+  bucket = var.file_transfer_bucket
+}
+
+data "aws_s3_bucket" "file_transfer_bucket_secure" {
+  bucket = var.file_transfer_bucket_secure
+}
+
+
+resource "aws_kms_grant" "file_transfer_encryption_key_grant" {
+  name              = "${var.environment}-file-transfer-service-key-grant"
+  key_id            = aws_kms_key.file_transfer_encryption_key.key_id
+  grantee_principal = aws_iam_role.file_transfer_service_execution.arn
+  operations        = ["Encrypt", "Decrypt", "GenerateDataKey"]
+}
```

Comment:
Once these have been moved I'd be tempted to rename this file kms.tf. We're using the bucket but not editing the bucket. Happy to hear alternatives